### ISSUE TITLE
'make' command fails due to missing catalog-v001.xml

### ISSUE THREAD

**Issue by lewismc** [cmungall/sweet-obo-alignment#3]
When a user downloads this project and attempt to ```make setup```, ```. env.sh``` and ```make``` the final task fails initially as catalog-v001.xml does not yet exist within sweet/src/ yet. This is a trivial bug which we should iron out.

**Comment by cmungall** [comment]
I'm thinking of making a Docker container to run this.. would that help?

**Comment by lewismc** [comment]
@cmungall yes I think it would... I mean in all honesty it is not too complicated to do however it may be easier for other folks if there was a container yes.
